### ISSUE TITLE
for bedrock please

### ISSUE THREAD

**Issue by IzsuPy** [thecoder-001/MineColab#81]
hey, is there a way to do it for bedrock too?

**Comment by broccoder** [comment]
Not really, since google colab doesn't support UDP, which is required for hosting bedrock servers. Isn't that right @thecoder-001 ?

**Comment by thecoder-001** [comment]
Yup, we would need a tunnel that supports UDP. playit.gg seems to support UDP, so I think its doable. Cloudflare argo might also support it, though I'm not sure.
Would try to add it whenever free. PRs welcome.
Cheers
